### DISCUSSION TITLE
MQTT Unavailable but Integration Works Fine

### DISCUSSION THREAD

**Discussion by efaden** [zwave-js/zwave-js-ui#3990] [General]
Having an issue where after a few days the entities through the mqtt integration go unavailable, but the direct home assistant integration entities work totally fine.  Restarting zwave js UI fixes it.  I didn't see any errors in the log or anything.  Anyone seen that?

**Comment by robertsLando** [comment]
First question is why are you using both mqtt discovery and integration? Coudl't you use just the integration?

**Comment by efaden** [comment]
Was actually using MQTT but it kept breaking.  Use MQTT from some non home
assistant scripts, node red etc.  But having it keep failing made me see if
the integration worked.

**Reply by robertsLando** [reply]
Do you see any issue on console that coould explain why the mqtt client is disconnecting?

**Comment by efaden** [comment]
Nothing.  That's why I'm really confused.  Didn't see any errors anywhere.

**Reply by robertsLando** [reply]
No clue so, what does makes mqtt unavailable on entities? Shouldn't that reflect node `/status` topic?
